common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -0,0 +1,67 @@
+@import common.NewNavigation
+
+@import common.TopLevelSection
+
+@sectionList(topLevelSection: TopLevelSection) = {
+    <li class="main-navigation__item navigation-border">
+            <!-- TODO: Firefox -->
+        <details>
+            <summary class="main-navigation__item__button">
+                @fragments.inlineSvg("arrow-down", "icon")
+                @topLevelSection.name
+            </summary>
+            <ul class="main-navigation__secondary navigation-group">
+                @NewNavigation.sectionItems(topLevelSection).map { sectionItem =>
+                    <li class = "navigation-group__item" ><a href="@sectionItem.url" tabindex="0">@sectionItem.name</a></li>
+                }
+            </ul>
+        </details>
+    </li>
+}
+
+<div class="main-menu-container" id="main-menu">
+        <!-- TODO: Provide tabbable alternative -->
+    <a href="#" class="main-menu-container__overlay"></a>
+
+    <div class="main-menu-container__menu">
+
+        <div class="main-menu-container__profile-container">
+            @fragments.inlineSvg("profile-36", "icon", List("rounded-icon", "main-menu-container__profile-icon"))
+            Sign in/up
+        </div>
+
+        <ul class="main-navigation">
+            @NewNavigation.topLevelSections.map { section =>
+                @sectionList(section)
+            }
+        </ul>
+        <ul class="navigation-group navigation-border">
+            <li class="navigation-group__item"><a href="#" tabindex="0">Jobs</a></li>
```

Comment:
Why are you adding `tabindex` here? By default, `A` is focusable and counted in the tab order. The only use case that I know of would be if you want to disable tab order but want programmatic focus with `tabindex="-1"`